controller/types.go

```diff
@@ -0,0 +1,23 @@
+package controller
+
+// mapLabels implements labels.Labels. See
+// https://godoc.org/k8s.io/apimachinery/pkg/labels#Labels.
```

Comment:
I couldn't find anything in the API that would convert map[string]string into labels.Labels. If you know about something or have suggestions LMK.

Reply:
It was labels.Set.